terraform/events.tf

```diff
@@ -0,0 +1,27 @@
+# This rule triggers the healthcheck lambda every minute
+resource "aws_cloudwatch_event_rule" "trigger_healthcheck" {
+    name                = "${local.domain_name_base[var.environment]}-healthcheck-trigger"
+    description         = "Fires the healthcheck lambda function every minute"
+    schedule_expression = "rate(1 minute)"
+}
+
+# Targets the healthcheck lambda
+resource "aws_cloudwatch_event_target" "sml_site_trigger_healthcheck" {
+    rule      = "${aws_cloudwatch_event_rule.trigger_healthcheck.name}"
+    target_id = "check_sml_site"
+    arn       = "${aws_lambda_function.healthcheck.arn}"
+    input     = jsonencode({
+                  "site"            : "https://${local.domain_name_base[var.environment]}",
+                  "env"             : "${var.environment}",
+                  "expected_string" : "An open source library for statistical code approved by the ONS"
```

Comment:
Can you make this a tf variable.  This makes it more flexible as we could potentially check different responses in different environments.

Reply:
Done